tests/rbd/rbd_utils.py

```diff
@@ -662,10 +665,9 @@ def rbd_bench(self, **kw):
         """
         cmd = f"rbd bench --io-type {kw.get('io_type', 'write')} --io-threads {kw.get('io_threads', 16)}"
         cmd += f" --io-total {kw.get('io', '500M')} {kw.get('imagespec')}"
-        return self.exec_cmd(
-            cmd=cmd,
-            long_running=True,
-        )
+        if kw.get("client_id"):
+            cmd += f" -n {kw['client_id']}"
+        return self.exec_cmd(cmd=cmd, **kw)
```

Comment:
Wouldn't it be better if we pass kw['client_id'] to exec_cmd and save line 668 and 669 in many places ?

Reply:
Done.